### ISSUE TITLE
failed to get the memory descriptor

### ISSUE THREAD

**Issue by lmenegazzo** [horosproject/horos#270]
Hello!!!
i`m having faults in my console related with HOROS only when scrolling images, and also feeling a little bit lag during that. 
The fault:

-static IOAccelSysMemory *IOAccelSysMemory::withClientBuffer(IOGraphicsAccelerator2 *, task_t, IOAccelShared2 *, mach_vm_address_t, uint64_t): failed to get the memory descriptor
- IOAccelSysMemory *IOAccelShared2::memory_with_client_buffer(mach_vm_address_t, uint64_t): withClientBuffer failed
lots of them when scrolling.

OSX 10.13.3 , HOROS 3.0 RC1, bur the faults also happened when 10.13 and HOROS 2.4.

Any ideas if this is HOROS or OSX related?

<img width="1735" alt="screen shot 2018-02-14 at 21 54 41" src="https://user-images.githubusercontent.com/36492767/36234738-e1c1d86e-11d3-11e8-96d5-0090eaa277c1.png">

**Comment by fvpolpeta** [comment]
Please, let me know if the same situation happens with the version 3.0.0 RC1, already available in our website.

Thanks

Fauze

**Comment by lmenegazzo** [comment]
Yes the same situation with 3.0.0 RC1, everything works fine , bur when scrolling the faults happen , a lot of then and feel the little lag also that is worse when exams have more images to scroll or when we sync two windows with different sequences.
